File: Training Problem 02/AC_solution.cpp
#include<bits/stdc++.h>  
using namespace std;

int main() {
    int N;
    cin >> N;
    for(int i = 0 ; i < N; ++i){
        int len,rep;
        string s = "", input;
        cin >> len >> input >> rep;
        for( int j = 0 ; j < rep ; ++j){
            s += input;
        }
        cout << s << endl;   
    }
    return 0;
}

File: TrainingProblem02/AC_solution.java
import java.util.Scanner;

public class AC_solution {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int N = scanner.nextInt();

        for (int i = 0; i < N; ++i) {
            int len, rep;
            String s = "", input;

            len = scanner.nextInt();
            input = scanner.next();
            rep = scanner.nextInt();

            for (int j = 0; j < rep; ++j) {
                s += input;
            }

            System.out.println(s);
        }

        scanner.close();
    }
}
